frontend/tests/e2e/search/features/work-flow-service-NegativeScenarios.feature

```diff
@@ -0,0 +1,149 @@
+  # 3. Required Field Validation – Empty Fields Block Progress
+  ###########################################################################
+  @negative @validation @workflow
+  Scenario: User attempts to proceed with empty required fields
+    Given the user is on a form step with required fields
+    When the user clicks Next without entering required information
+    Then progress is blocked
+    And error messages appear for each required field
+
+  ###########################################################################
+  # 4. Cross Form Required Validation – Missing Required Data at Submission
+  ###########################################################################
+  @negative @validation @submission @workflow
+  Scenario: Submission fails due to missing answers from previous steps
+    Given the user skipped a required question in an earlier step
+    When the user clicks Submit on the Review & Submit page
+    Then submission fails
+    And the user is redirected to the step with missing data
+    And an error message is displayed
+
+  ###########################################################################
+  # 5. Successful Submission – Backend Submission Error
+  ###########################################################################
+  @negative @submission @workflow
+  Scenario: Submission fails due to backend error
+    Given the user has completed all required fields
+    When the user clicks Submit
+    Then the system displays "Submission failed"
+    And the user stays on the review page
+
+  ###########################################################################
+  # 6. Data Persist on Refresh – Data Loss After Refresh
+  ###########################################################################
+  @negative @persistence @workflow
+  Scenario: Data does not persist after page refresh
+    Given the user enters information on a form step
+    When the user refreshes the page
+    Then the entered data is missing
+    And the user sees a message indicating data was lost
+
+  ###########################################################################
+  # 7. Save and Continue Later – Save Fails
+  ###########################################################################
+  @negative @saving @workflow
+  Scenario: Save and Continue Later fails
+    Given the user has entered progress in the application
+    When the user selects "Save and Continue Later"
+    Then the system displays "Unable to save your progress"
+    And a draft is not created
+
+  ###########################################################################
+  # 9. File Upload – Invalid File Rejected
+  ###########################################################################
+  @negative @upload @workflow
+  Scenario: User uploads invalid file type
+    Given the user is on the file upload step
+    When the user uploads a ".exe" file
+    Then the system rejects the file
+    And the user sees an error "Invalid file type"
```

Comment:
this isn't enforced at the moment, can we flag this as something to revisit later?